api/resolvers/wallet.js

```diff
@@ -353,7 +347,7 @@ const resolvers = {
   },
   WalletDetails: {
     __resolveType (wallet) {
-      return wallet.address ? 'WalletLNAddr' : wallet.macaroon ? 'WalletLND' : 'WalletCLN'
+      return wallet.address ? 'WalletLNAddr' : wallet.macaroon ? 'WalletLND' : wallet.rune ? 'WalletCLN' : 'WalletLNbits'
```

Comment:
there is probably a way to generate this code

Reply:
The 3rd parameter passed to [`__resolveType`](https://www.apollographql.com/docs/apollo-server/schema/unions-interfaces/#resolving-a-union) is this [info object](https://github.com/graphql/graphql-js/blob/a24a9f35b876bdd0d5050eca34d3020bd0db9a29/src/type/definition.ts#L891-L902). It might have the grandparent in it to determine the type.